sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -317,7 +317,13 @@ export interface IssuerProperties {
 }
 
 // @public
-export type KeyUsageType = 'digitalSignature' | 'nonRepudiation' | 'keyEncipherment' | 'dataEncipherment' | 'keyAgreement' | 'keyCertSign' | 'cRLSign' | 'encipherOnly' | 'decipherOnly';
+export type KeyCurveName = string;
+
+// @public
+export type KeyType = string;
```

Comment:
KeyType was previously not part of the public API. Why are we adding it?

Reply:
- KeyType(JsonWebKeyType) is used as part of KeyProperties object. 
  - This KeyProperties is used as part of CertificatePolicy. 
  - The CertificatePolicy is used as part of ImportCertificateOptions. 
  - The ImportCertificateOptions is one of the parameters for importCertificate API.

So, if the user wants to construct the importCertificate, we need to expose this value. To my best guess, it should have been done already. Not sure why it was ignored?

Reply:
I found the reason why it was ignored. It was exported as CertificateKeyType. I will sync up with you offline and decide on this.

Reply:
Removed it (in the next commit)